Dockerfile

```diff
@@ -1,35 +1,25 @@
-FROM centos:7
+FROM alpine:3.3
 
-ENV TERRAFORM_VERSION 0.6.10
-ENV TERRAFORM_STATE_ROOT /state
+RUN apk add --no-cache build-base git openssh openssl py-pip python python-dev unzip \
+	&& git clone https://github.com/CiscoCloud/mantl /mantl \
```

Comment:
This will only update the files at image build time; they should be updated at run time.